avaframe/com1DFA/glideSnowCfg.ini

```diff
@@ -0,0 +1,47 @@
+### Config File - This file contains the main settings for the glide snow simulation run
+## Copy to  local_glideSnowCfg.ini and set you parameters
+# This file is part of Avaframe.
+
+
+[GENERAL]
+# so far no general parameters to be set as glide snow option is fully implemented in com1DFA
+# hence checkout the com1DFA_override section
+
+
+[com1DFA_override]
+# use default com1DFA config as base configuration (True) and override following parameters
+# if False and local_com1DFACfg is available use local
+defaultConfig = True
```

Comment:
The reason why I put it here is to make it analog to the other ini files we have where there is an override section. I think it is almost required once there is more than one override section in the ini, so that it can be used differently for different modules